content/docs/latest/agentgateway/agent/about.md

```diff
@@ -51,17 +51,21 @@ sequenceDiagram
     Agent->>+LLM: Forward tool result
     LLM-->>+Agent: Generate final response
     Agent-->>-User: "The weather in Columbus is 15°C and cloudy."
-```
-
+    ```
 1. The User asks about the weather today. The request includes a list of tools that the LLM can use to answer the question.
-2. The Agent sends the request with the tool list to the LLM.
-3. Based on the conversation, the LLM suggests to call the `get_weather` tool and includes the required parameters to use that tool. The LLM might get these parameters by using the chat history, asking the end user to provide them, or by suggesting other tools to call before calling the `get_weather` function.
-4. The Agent invokes the `get_weather` tool by sending a request to the App. The request includes the required parameters to execute the tool.
-5. The App executes the `get_weather` tool with the required parameters to get the current weather in the location, such as Columbus, OH.
-6. The App returns the tool result to the Agent.
-7. The Agent returns the tool result to the LLM.
-8. The LLM uses the real-time tool result from the App in a natural language response and returns this response to the Agent.
-9. The Agent then forwards the natural language response to the User.
 
+2. The agents sends the request with the tool list to the LLM.
```

Comment:
can you revert all the changes to these steps? it looks like it replaced the terms and markdown formatting, and has some subject-verb agreement issues.